hazelcast/src/test/java/com/hazelcast/nio/ConnectionTest.java

```diff
@@ -65,7 +65,7 @@ public void cleanup() throws Exception {
     }
 
     @Test
-    @Ignore
+//    @Ignore
```

Comment:
The whole class is ignored, so I don't know if this will change anything. The class is also missing `@RunWith` and `@Category`:
```
@Ignore("See testBlockedClientSockets and testBlockedClientSockets2 tests. " +
        "Currently we couldn't find a way to make them pass...")
public class ConnectionTest {
```